content/handbook/engineering/infrastructure-platforms/gitlab-delivery/delivery/_index.md

```
@@ -17,302 +17,306 @@ stale: false
-1. `Discussion` - 作業エピックに昇格するか、別の実装 Issue を生み出す可能性のあるメタ Issue
-1. `Announcements` - より広いオーディエンスへの重要な変更を告知するために使用される Issue
+1. `onboarding` - Issue はチームリソースへのアクセス付与に関するものです。
+1. `team-tasks` - 一般的なチームトピックに関する Issue。
+1. `Discussion` - 作業中のエピックに昇格する可能性が高い、または別の実装 Issue を生み出す可能性が高い meta Issue。
+1. `Announcements` - 重要な変更を広範な視聴者にアナウンスするために使用される Issue。
 
-プロジェクトラベルは必要に応じて定義されますが、Issue がチーム管理タスクを説明する場合を除いて必須です。
+プロジェクトラベルは必要に応じて定義されますが、Issue がチーム管理タスクを記述している場合を除き、必要です。
 
-Delivery に影響するインシデントにはオプションで[インパクトラベル](#delivery-impact-labels)を含めることができます。
+Delivery に影響するインシデントには、[impact ラベル](#delivery-impact-labels) が任意で含まれる場合があります。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)